client/src/components/graphs/LikertScale.js

```diff
@@ -89,34 +108,99 @@ const LikertScale = ({ onChange, value, levels, width, height, readonly }) => {
           </React.Fragment>
         )
       })}
+      {values?.map((xValue, index) => (
+        <g
+          transform={`translate(${scale(xValue)} ${scaleYPosition})`}
+          key={`values-${index}-${xValue}`}
+        >
+          <path
+            d="M -10,-10 L 10,10 M 10,-10 L -10,10"
+            style={{
+              stroke: "black",
+              fill: "black",
+              strokeWidth: 2
+            }}
+          />
+        </g>
+      ))}
+
+      {values?.length > 1 && (
+        <g transform={`translate(0 ${scaleYPosition})`}>
+          <line
+            x1={scale(valuesStats.min)}
+            y1="-15"
+            x2={scale(valuesStats.min)}
+            y2="15"
+            style={{ stroke: "black", strokeWidth: 3 }}
+          />
+          <line
+            x1={scale(valuesStats.max)}
+            y1="-15"
+            x2={scale(valuesStats.max)}
+            y2="15"
+            style={{ stroke: "black", strokeWidth: 3 }}
+          />
+          <line
+            x1={scale(valuesStats.min)}
+            y1="0"
+            x2={scale(valuesStats.max)}
+            y2="0"
+            style={{ stroke: "black", strokeWidth: 3 }}
+          />
+        </g>
+      )}
+
+      {values?.length > 0 && (
+        <g transform={`translate(0 ${scaleYPosition})`}>
+          <circle
+            cx={scale(valuesStats.avg)}
+            r="10"
```

Comment:
Shouldn't these all use `{-15}` etc instead of `"-15"`, at least for consistency?

Reply:
done